### PR TITLE
[Docs] Linux documentation for local setup

### LINKED ISSUE #10940: ✏️ No-maintenance-container setup instructions
## ✏️ Documentation

Some members of the team have found it useful to run composer and pnpm commands directly in their host operating system rather than using the maintenance container.  Let's document that setup.

## ✅ Acceptance Criteria

- [ ] New documentation in the repo for this setup

### PR DESCRIPTION
**Pull request by petertgiles**
🤖 Resolves #10940

## 👋 Introduction

This branch adds some documentation for setting up a local Linux or WSL environment with all the tooling need to run the app.

## 🧪 Testing

Spelling and grammar?  Confusing spots?  Suggestions for clarity and organization?   Factual errors?

### CODE REVIEW

File: documentation/linux-setup.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+```
+groups
+```
+
+## Hosts file
+
+We need to ensure that scripts running locally can find the services running in Docker.
+
+```
+sudo nano /etc/hosts
```
Comment (esizer) [inline documentation/linux-setup.md:54]: Nano?! :exploding_head:

Comment (petertgiles) [reply documentation/linux-setup.md:54]: Sadly, Neovim isn't in the default install.  :laughing:

File: documentation/linux-setup.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+
+Double check:
+
+```
+composer --version
+```
+
+It should be major version 2.
+
+### Node.js
```
Comment (mnigh) [inline documentation/linux-setup.md:135]:
```suggestion
## Node.js
```

File: documentation/linux-setup.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+> [!IMPORTANT]
+> The install script will also instruct you to run a `source` command to activate the tool in your session.
+
+Double check:
+
+```
+pnpm --version
+```
+
+## Make File
```
Comment (mnigh) [inline documentation/linux-setup.md:185]:
```suggestion
## Makefile
```

**Review by mnigh — CHANGES_REQUESTED** [review]

**Review by mnigh — APPROVED** [review]
I can't really test but seems fine 🤷

**Pull request merged**